docs/dev/relay_add_pass.rst

```diff
@@ -0,0 +1,125 @@
+.. _relay-add-pass:
+
+Adding a Compiler Pass to Relay
+===============================
+
+Compiler passes can be used to collect data about Relay programs or transform
+them in various ways. The base class used to traverse programs is
+``ExprFunctor``. Depending on what you want to do, there are subclasses
```

Comment:
I recommend using `:code:` for class names and other such constructs, as using the backticks without `:code:` results in italics rather than typewriter font.